server/controllers/createLocation/cellDetails.ts

```diff
@@ -0,0 +1,31 @@
+import FormWizard from 'hmpo-form-wizard'
+import { NextFunction, Response } from 'express'
+import FormInitialStep from '../base/formInitialStep'
+import backUrl from '../../utils/backUrl'
+import { TypedLocals } from '../../@types/express'
+
+export default class CellDetails extends FormInitialStep {
+  middlewareSetup() {
+    super.middlewareSetup()
+  }
+
+  locals(req: FormWizard.Request, res: Response): Partial<TypedLocals> {
+    const locals = super.locals(req, res)
+    const { locationId, prisonId, decoratedResidentialSummary } = res.locals
+    const localName = req.sessionModel.get<string>('localName')
+    const locationName = localName || req.sessionModel.get<string>('locationCode')
```

Comment:
```suggestion
    const locationName = localName || `${decoratedResidentialSummary.location.pathHierarchy}-${req.sessionModel.get<string>('locationCode')}`
```

Reply:
I believe that if a local name is set, it should be displayed by itself, rather than `A-1-localname`